WireAnalytics/Sources/WireAnalytics/Events/Predefined/AnalyticsEvent+Calling.swift

```diff
@@ -30,13 +30,14 @@ public extension AnalyticsEvent {
 
         public static func initiatedCall(
             isVideo: Bool,
-            conversationType: ConversationType
+            conversationType: Segmentation.Conversation.ConversationType
         ) -> AnalyticsEvent {
             AnalyticsEvent(
                 name: "calling.initiated_call",
                 segmentation: [
                     .isVideoCall(isVideo),
-                    .groupType(conversationType)
+                    .conversationType(conversationType),
+                    .Removed.groupType(conversationType.mapToConversationType())
```

Comment:
What's the difference between these two?

Reply:
`group_type` is an old key. There is a comment in the ticket:

"... other events might rely on `group_type` so I would suggest to add a renamed copy of it until the other events are switched to `conversation_type`"
https://wearezeta.atlassian.net/browse/WPB-12199?focusedCommentId=132080

I left the link near the declaration and deprecation.